working/augmentation-libraries/feature-specification.md

```diff
@@ -721,16 +729,85 @@ It is a compile-time error if:
 *   The constructor augmentation specifies any default values. *Default values
     are defined solely by the original constructor.*
 
-*   The original constructor is `const` and the augmenting constructor is not
+*   The augmented constructor is `const` and the augmenting constructor is not
     or vice versa.
 
-*   The original constructor is a factory constructor and the augmenting
-    constructor has an initializer list.
+*   The augmented constructor is a factory constructor and the augmenting
+    constructor is not or vice versa.
+
+*   The resulting constructor is not valid (has a redirecting initializer and
+    other initializers, multiple `super` initializers, etc).
+
+#### Non-redirecting generative constructors
+
+These are probably the most complex constructor, but also the most common.
+
+A non-redirecting generative constructor marked `augment` may:
+
+*   Add or replace the body of the existing constructor with a new body.
+
+    *   If the augmenting constructor has a body (a block or => body, not just
+        a single ;), then that body replaces the existing constructor body.
```

Comment:
Should code quote the `=>` and `;`. Especially the `;`, because ";)" looks like a winky-smiley! :wink: